backend/molgenis-emx2-sql/src/test/java/org/molgenis/emx2/sql/TestValidationExpressions.java

```diff
@@ -0,0 +1,26 @@
+package org.molgenis.emx2.sql;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.molgenis.emx2.Column;
+import org.molgenis.emx2.Row;
+import org.molgenis.emx2.utils.TypeUtils;
+
+public class TestValidationExpressions {
+
+  @Test
+  public void testValidationExpressions() {
+    String exp = "{birthYear} <= {currentYear}";
```

Comment:
expand tests?

Reply:
Is this the same as TestValidation.java? It feels like this test should be done in the expressions repository.